src/choices/src/components/styled/ChoiceChipStyles.js

```diff
@@ -0,0 +1,61 @@
+import { css } from 'styled-components';
+
+import { NEUTRAL, PURPLE } from '../../../../colors';
+
+// Not really sure why, but 1px box-shadow looks significantly larger than 1px on chrome
+// Using fractional pixel seems to look closer to intended design
```

Comment:
is this just on chrome or is this just comparing to the designs? could be figma applying some antialiasing or some scaling